apps/io-wallet-user-func/src/infra/http/handlers/create-wallet-instance.ts

```diff
@@ -31,6 +33,22 @@ type WalletInstanceRequestPayload = t.TypeOf<
   typeof WalletInstanceRequestPayload
 >;
 
+const sendEmail =
+  (
+    fiscalCode: string,
+  ): RTE.ReaderTaskEither<
+    { emailQueuingEnabled: boolean; queueClient: QueueClient },
+    Error,
+    void
+  > =>
+  ({ emailQueuingEnabled }) =>
+    emailQueuingEnabled
+      ? pipe(
+          TE.tryCatch(async () => enqueue(fiscalCode), E.toError),
```

Comment:
`enqueue(fiscalCode)` is a `ReaderTaskEither`

Reply:
you can do something like
```
const foo: (
  fiscalCode: FiscalCode,
) => RTE.ReaderTaskEither<
  { emailQueuingEnabled: boolean; queueClient: QueueClient },
  Error,
  void
> =
  (fiscalCode) =>
  ({ emailQueuingEnabled, queueClient }) =>
    emailQueuingEnabled
      ? pipe({ queueClient }, enqueue(fiscalCode))
      : TE.right(undefined);
```
and delete the feature flag on the `sendEmail` method in `EmailNotificationServiceClient`